extensions/amazon-lambda-http/runtime/src/main/java/io/quarkus/amazon/lambda/http/LambdaHttpHandler.java

```diff
@@ -167,8 +167,16 @@ private APIGatewayV2HTTPResponse nettyDispatch(InetSocketAddress clientAddress,
         quarkusHeaders.setContextObject(Context.class, context);
         quarkusHeaders.setContextObject(APIGatewayV2HTTPEvent.class, request);
         quarkusHeaders.setContextObject(APIGatewayV2HTTPEvent.RequestContext.class, request.getRequestContext());
+        HttpMethod httpMethod = null;
+        if (request.getRequestContext() != null && request.getRequestContext().getHttp() != null
+                && request.getRequestContext().getHttp().getMethod() != null) {
+            httpMethod = HttpMethod.valueOf(request.getRequestContext().getHttp().getMethod());
+        }
+        if (httpMethod == null) {
+            throw new IllegalStateException("Missing http method in request event!");
```

Comment:
```suggestion
            throw new IllegalStateException("Missing HTTP method in request event");
```
Minor nitpicking, sorry :). Could you adjust and squash your commits? Thanks.